Database/Corrections/sodQuestFixes.lua

```diff
@@ -2617,6 +2617,18 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.requiredSpell] = -425216,
             [questKeys.zoneOrSort] = sortKeys.PRIEST,
         },
+        [90161] = {
+            [questKeys.name] = "Strength of Soul",
+            [questKeys.startedBy] = {{3924,3750,3922,3748},{450003},{211534}},
+            [questKeys.finishedBy] = nil,
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 23,
+            [questKeys.requiredRaces] = raceIDs.NONE,
+            [questKeys.requiredClasses] = classIDs.PRIEST,
+            [questKeys.objectivesText] = {"Kill Thislefur for Primal Insight, then go up in to the tree, and use it whilst near the Thislefur Dreamcatcher, and receive the rune."},
```

Comment:
You need to have two mediation buffs active.

Reply:
Fixed in: https://github.com/Questie/Questie/pull/5492/commits/259891f40e236ff011396fe9821b18be3fc574f4